src/Infrastructure/Service/FinderFactory.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Utils\Infrastructure\Service;
+
+use Symfony\Component\Finder\Finder;
+
+class FinderFactory
```

Comment:
Looks like an obsolete factory, isn't it?

Reply:
Removed it